### ISSUE TITLE
JOSS Review: Example Script

### ISSUE THREAD

**Issue by abhiramm7** [julianreul/mode_behave#6]
Hi, 

Consider updating the `example_estimation.py` script with visualization examples. I might have missed something, but I couldn't find an example of how to use the visualization functionality in the package. I would also recommend adding the script used to generate the figures presented in the manuscript.

Thanks,
Abhi

**Comment by julianreul** [comment]
Hi,

thanks again for your comments!

I added visualization routines to the example_estimation.py in an earlier update in response to the comments by Leander Kotzur. The script includes two visualization methods:
1) .visualize_space(): This method visualizes the distribution of the estimated choice preferences across the base population.
2) .forecast(): This method visualizes the simulated choice probabilities for different consumer groups and can be additionally used to conduct sensitivity analyses. 
All generated visualizations are stored in the folder .\mode_behave_public\Visualizations\, whereas "." indicates the location of the package's home directory.

Within the script example_estimation.py, the lines 91-96 (1st call of .forecast()) generate Figure 2 of the paper, whereas lines 104-110 (2nd call of forecast()) generate Figure 1 of the paper. Additionally, I added cross-references in the paper to refer to the script example_estimation.py and vice versa.

I hope to have addressed your comments sufficiently.

Best 
Julian
